activities.el

```diff
@@ -497,7 +498,8 @@ accordingly."
 (defun activities-mode--killing-emacs ()
   "Persist all activities' states.
 To be called from `kill-emacs-hook'."
-  (let ((activities-always-persist t))
+  (let ((activities-always-persist t)
+        (activities-anti-save-predicates nil))
```

Comment:
I'm not sure this is a good idea.  The idea of the anti-save-predicates is that, if any of them return non-nil, some condition is such that the activity should not be saved (e.g. it might mean that saving would fail, or that trying to restore the activity from that state would fail).  That Emacs is exiting is not necessarily a reason to ignore that condition.  I think we shouldn't do this unless we know a good reason to make this exception; and even then, it might be better to make it optional, or to suggest some way to do the check within each predicate.

Reply:
I figured that would be a bit controversial, I will remove next iteration.